ConnectivityTestScriptsForWindows/Publish-Report.ps1

```diff
@@ -51,9 +51,22 @@ $AADCommonIssue =
 <div style='background-color:#FF033E;color:white;padding:20px;'>
 <h2>Azure Active Directory (AAD) Common Issue</h2>
 <ul>
-<li> Check resolution for AAD FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China)</li>
-<li> Check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to Service Tag AAD <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags</a></li>
-Check firewall settings for outbound connection to URLs mentioned under sections 56 and 59 in <a href='https://docs.microsoft.com/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true#microsoft-365-common-and-office-online'> Microsoft 365 Common and Office Online .</a> </li>
+<li> Check DNS resolution for Azure active directory FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China). 
+</br>
+Run 
+</br>
+Resolve-DnsName login.microsoftonline.com
+</br> (or equivalent FQDN for the Azure environment in powershell) on the machine and verify if you are able to resolve the uri. </li>
+<li> Check Network Security Group (NSG) attached to virtual machine's subnet, NSG should allow outbound connection to AzureActiveDirectory <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags</a>
+</br>
+Network Security Groups for a virtual machine can be checked at Azure portal Virtual machine blade -> Networking Settings -> Network Interface -> Outbound port rules. <IMG SRC='media/VMNetworkblade.png' ALT='Virtual machine NSG outbound port rules' width=1024 height=480/>
+</br>
+If the rule is missing, Click on 'Add outbound port rule' and fill in the source as Virtual network, destination as Service tag and Destination Service tag as AzureActiveDirectory. Set Action as Allow and click Add.
```

Comment:
'Service tag' and 'AzureActiveDirectory'? (within quotes)

Reply:
Changed